### PR TITLE
Port #803 - web zoom toggle

### PR DESCRIPTION
**Pull request by rpdome**

### CODE REVIEW

File: common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/AuthorizationFragment.java [hunk 1]
```diff
@@ -97,11 +97,15 @@
     @VisibleForTesting
     static final String KEY_REQUEST_HEADERS = "requestHeaders";
 
+    static final String WEB_VIEW_ZOOM_CONTROLS_ENABLED = "com.microsoft.identity.web.view.zoom.controls.enabled";
```
Comment (iambmelt) [inline common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/AuthorizationFragment.java]: Duplicated constant?

File: common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/AuthorizationFragment.java [hunk 2]
```diff
@@ -97,11 +97,15 @@
...
+
+    static final String WEB_VIEW_ZOOM_ENABLED = "com.microsoft.identity.web.view.zoom.enabled";
```
Comment (iambmelt) [inline common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/AuthorizationFragment.java]: Duplicated constant?

Comment (shahzaibj) [reply common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/AuthorizationFragment.java]: They are different

Comment (iambmelt) [reply common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/AuthorizationFragment.java]: Duplicated across files, see `AuthorizationActivity.java`

**Pull request merged**